sql/service/src/main/scala/org/apache/spark/sql/service/cli/SparkSQLCLIService.scala

```diff
@@ -45,10 +45,10 @@ private[service] class SparkSQLCLIService(
 
   override def init(sqlConf: SQLConf): Unit = {
     setSuperField(this, "sqlConf", sqlConf)
+    val sessionManager = new SessionManager(sparkServer, sqlContext)
+    setSuperField(this, "sessionManager", sessionManager)
```

Comment:
could we by the way get rid of all this reflection stuff just by changing the super field to protected?

Reply:
> could we by the way get rid of all this reflection stuff just by changing the super field to protected?

fi na l lu